packages/salesforcedx-vscode-core/src/commands/auth/forceAuthAccessTokenLogin.ts

```diff
@@ -46,8 +52,10 @@ export class ForceAuthAccessTokenExecutor extends LibraryCommandletExecutor<
       await authInfo.save();
       await authInfo.setAlias(alias);
       await authInfo.setAsDefault({
-        defaultUsername: true
+        org: true,
+        devHub: true
       });
+      // await authInfo.handleAliasAndDefaultSettings(sideEffects)
```

Comment:
Can remove this

Reply:
Actually we inverted this! Wanted to use handleAliasAndDefaultSettings to handle everything nicely and only call save once.